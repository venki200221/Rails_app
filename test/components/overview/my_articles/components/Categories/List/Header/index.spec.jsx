import React from 'react';
import { shallow } from 'enzyme';
import '../../../../../../../testHelper';

import Header from '@components/overview/my_articles/components/Categories/List/Header/Header.jsx';

describe('Header', () => {
  it('renders the title, message, and tooltip', () => {
    const component = shallow(
      <Header message="message" sub="subtext" title="title" />
    );
    expect(component).toMatchSnapshot();
  });

  it('renders just the title if there is no message for the tooltip', () => {
    const component = shallow(<Header title="title" />);
    expect(component).toMatchSnapshot();
  });
});
