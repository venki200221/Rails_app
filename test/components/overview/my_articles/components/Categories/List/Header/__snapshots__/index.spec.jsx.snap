// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders just the title if there is no message for the tooltip 1`] = `ShallowWrapper {}`;

exports[`Header renders the title, message, and tooltip 1`] = `ShallowWrapper {}`;
